projects/batfish/src/org/batfish/grammar/cisco/Cisco_logging.g4

```diff
@@ -106,8 +109,8 @@ logging_host
 :
    HOST iname = variable? hostname = variable
    (
```

Comment:
I could find examples for VRF and Discriminator order; this can be generalized with a Kleene's closure, but then it will accept many sentences that are not legal (e.g. repeating VRF clauses, etc.) nor practical.